common/app/common/Box.scala

```diff
@@ -0,0 +1,46 @@
+package common
```

Comment:
This is a copypasta from https://github.com/guardian/box/blob/master/src/main/scala/com/gu/Box.scala